src/resolver/resolver.js

```diff
@@ -222,7 +222,7 @@ class Resolver {
       ? this.constructor.__createUrl(
         this.baseUrl,
         document.baseURI || document.URL
-      ).href.replace(/[^\/]*$/, '')
```

Comment:
This was an unnecessary escape (complained by linter rule). Most characters don't need to be escaped inside a character group `[]`.